src/sidebar/tab/tab.js

```diff
@@ -137,7 +137,7 @@ export default class ContainerTab {
             browser.contextualIdentities
                 .get(this.tab.cookieStoreId)
                 .then((ci) => {
-                    this.elements["link"].style.borderBottomColor = ci.colorCode
+                    this.elements["link"].style.borderTopColor = ci.colorCode
```

Comment:
This might break custom CSSS that people have applied in the extension (if they've modified the border).

Instead of changing this for all themes, it could be solved by setting the css variable via javascript for this specific element.

`this.elements["link"].style.setProperty('--color-default-container', ci.colorCode);`

Reply:
On a second thought, that assumes that people use the css variable in their custom css, which is not necessarily the case, so maybe this is worse...

Reply:
I think we should just set `borderColor` so that the side on which the border appears is customizable from CSS.

I also like the way the top border looks now, so great work on that front :clap: .